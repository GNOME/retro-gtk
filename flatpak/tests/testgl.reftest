[Retro Reftest]
Path=/testgl
Core=/app/lib/libretro/testgl_libretro.so

[Options]
testgl_resolution=320x240;360x480;480x272;512x384;512x512;640x240;640x448;640x480;720x576;800x600;960x720;1024x768;1024x1024;1280x720;1280x960;1600x1200;1920x1080;1920x1440;1920x1600;2048x2048;

[Frame 0]
State=None
