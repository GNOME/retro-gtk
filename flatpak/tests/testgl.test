[Retro Reftest]
Path=/testgl
Core=/app/lib/libretro/testgl_libretro.so

[Frame 0]
Tests=Run;StateRefresh;
