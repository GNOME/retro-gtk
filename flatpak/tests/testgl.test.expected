/testgl/Boot: OK
/testgl/0/Run: OK
/testgl/0/StateRefresh: OK
